src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,14 +29,16 @@
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
-
+  private final BallChucker9000 BallChucker9000Subsystem;
 
   /**
    * The container for the robot.  Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
     // Configure the button bindings
     configureButtonBindings();
```

Comment:
By the way, here's the buttons.
manipulator   -- left bumper: manual rotator left.
                      right bumper: manual rotator right.
Right joystick -- trigger: shoot
                         Button 3: indexer forward
                         Button 2: indexer backward